Source/WebKit/WebProcess/InjectedBundle/API/glib/WebKitWebPage.cpp

```diff
@@ -601,7 +603,11 @@ static void webkit_web_page_class_init(WebKitWebPageClass* klass)
         g_signal_accumulator_true_handled, nullptr,
         g_cclosure_marshal_generic,
         G_TYPE_BOOLEAN, 2,
+#if ENABLE(CONTEXT_MENUS)
         WEBKIT_TYPE_CONTEXT_MENU,
+#else
+        G_TYPE_OBJECT,
+#endif // ENABLE(CONTEXT_MENUS)
```

Comment:
I'm not sure if the introspection scanner can handle this or not. Let's give it a try.

Reply:
An alternative would be to just disallow `--no-context-menus` with the GTK port. Let me know if I should pursue that. (But if this `#else` doesn't cause problems, I'd prefer to do that in a separate PR)

Reply:
Well, the EWS are happy, so it's probably fine.